### PR TITLE
Add My Recipes page structure and styling

### LINKED ISSUE #1: Nav  Bar
- Brighter Color - Dark green to match Thyme photo
- Light text color
- Box Shadow
- Includes: Home,  Saved Recipes, Shopping Cart
- Present in all HTML files

### LINKED ISSUE #4: My Recipes
- Nav Bar
  - Home, saved recipes, shopping cart
- On one side: Recipes to Cook
  - Just display title
  - ability to click on it and have it take you to the recipe details

- On the other side: Favorite Recipes
  - search bar to filter by ingredient/title
  - listed by category tag
  - Just display title

### PR DESCRIPTION
**Pull request by zwsaile**
#### What's this PR do?
- Adds My Recipe page HTML structure and CSS styling

#### Where should the reviewer start?
- Line 19 in styles.css and line 20 in index.html

#### What are the relevant tickets?
- My Recipe tickets

#### Questions:
- Will this cause a million merge conflicts? We will see.

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -5,13 +5,19 @@
   /* border: solid 1px black; */
 }
 
+*:focus {
```
Comment (sted1994) [inline src/styles.css:8]: What does this do?

File: src/styles.css [hunk 2]
```diff
@@ -5,13 +5,19 @@
...
+  outline: none;
+}
+
 body {
   background-image: url("images/thyme.jpeg");
   background-size: cover;
   background-repeat: no-repeat;
```
Comment (sted1994) [inline src/styles.css:16]: Can you take this empty line out next push please

Comment (Wperricone) [inline src/styles.css:16]: I don't think you need this blank line...

**Review by sted1994 — APPROVED** [review]
Looks good! nice Work

**Review by Wperricone — APPROVED** [review]
Looking Great!

**Pull request merged**